packages/grafana-ui/src/components/ButtonCascader/ButtonCascader.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+
+// @ts-ignore
+import RCCascader from 'rc-cascader';
+
+export interface ButtonCascaderOption {
+  label: string;
+  value: string;
+
+  children?: ButtonCascaderOption[];
+  disabled?: boolean;
+  // Undocumented tooltip API
+  title?: string;
+}
+
+export interface ButtonCascaderProps {
+  options: ButtonCascaderOption[];
+  buttonText: string;
+  disabled?: boolean;
+  expandIcon?: React.ReactNode;
```

Comment:
Do we need to have the icon configurable? I think we should have the same icon used always